billsplit/src/components/Invoice.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+
+
+const Invoice = (props) => {
+    return (
+        <>
+            <div className="ui raised segment container">
+                <div className="ui large header">
+                    For the event of {props.eventTitle}
+                </div>
+                <div className="ui header">
+                    Your total is ${props.amountTotal}
+                </div>
+                <div className="ui header">
+                    With a party of {props.numberOfPartyMembers}
+                </div>
+                <div className="ui header">
+                    {props.taxPercent ? `Tax {props.taxPercent}%` : null}
```

Comment:
You just forgot the `$` in front of the opening curly brace in your string literal. This is why we are not getting the input value here.